core/src/main/java/org/infinispan/transaction/xa/CacheTransaction.java

```diff
@@ -130,7 +130,8 @@ default void putLookedUpRemoteVersion(Object key, EntryVersion version) {}
     * <p/>
     * Note: used in Repeatable Read + Write Skew + Clustering + Versioning.
     */
-   void replaceVersionRead(Object key, EntryVersion version);
+   @Deprecated
+   default void replaceVersionRead(Object key, EntryVersion version) { addVersionRead(key, version);}
```

Comment:
I wouldn't put the whole method on one line, not even if it's deprecated :)
Please also add a `@deprecated Since 9.0` in the javadoc, it'll make it easier to decide when to remove it later.